src/sentry/event_manager.py

```diff
@@ -671,6 +671,12 @@ def save(self, project, raw=False):
                 project=project,
                 sender=EventManager,
             )
+
+            metrics.incr(
+                'event.discarded',
+                skip_internal=True,
+                tags={'organization_id': project.organization_id},
```

Comment:
@JTCunning Any preference on this tag name? Pretty sure this is what we use elsewhere. Or should we use `instance` instead? I still don't know when we should use which.